docs/CONTRIBUTING.md

```diff
@@ -133,6 +135,36 @@ $ kubectl -n=tidb-operator-e2e logs -f tidb-operator-e2e
 
 To re-run e2e tests, delete the testing pod and apply it again.
 
+##### Run e2e tests in kind 
+
+Execute the follow command to create a [kind](https://kind.sigs.k8s.io/) Kubernetes environment.  
+
+```sh
+$ ./hack/kind-cluster-v1.12.8.sh
+```
+
+Then you can build and push Docker images to the public Docker registry (There have not docker registry in Kind Kubernetes environment currently)
```

Comment:
Docker images can be loaded with `kind load docker-image` or `kind load image-archive` command.

Reply:
this can push images to all nodes, but if `ImagePullPolicy` is `Always`, the pod still can not get image from local.  it can not resolve the image problem completely

Reply:
I think `kind load` also load image to all nodes. For image always pull issue, can we use `IfNotPresent` to replace `Always`.